artist/turtle/init.lua

```diff
@@ -20,8 +20,8 @@ return function(context)
       -- Scan all slots and attempt to determine which ones should be considered "protected"
       -- Namely, which ones shouldn't we pick up from.
       for i = 1, 16 do
-        local info = turtle.getItemDetail(i)
-        if info and info.name == item.meta.name and info.damage == item.meta.damage then
+        local info = turtle.getItemDetail(i,true)
+        if info and info.name == item.meta.name and info.nbt == item.meta.nbt then
```

Comment:
I think it's just probably easier to just not care about NBT here. This way we can do the scan in 0 ticks.

```suggestion
        local info = turtle.getItemDetail(i)
        if info and info.name == item.meta.name then
```